k8s/crd-samples/pool-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorpools.openebs.io
+  namespace: default
+spec:
+  serviceAccountName: openebs-maya-operator
+  group: openebs.io
+  names:
+    kind: CstorPool
```

Comment:
CStorPool (camel case). Apply this comment in all the occurances

Reply:
Okay will change. But CstorPool is came case right?

Reply:
C, Stor and Pool are different words.

Reply:
ok